R/stream_shiny.R

```diff
@@ -0,0 +1,67 @@
+#' Reactive spark reader
+#'
+#' Given a spark object, returns a reactive data source for the contents
+#' of the spark object. This function is most Useful to read Spark streams.
+#'
+#' @param x An object coercable to a Spark DataFrame.
+#' @param intervalMillis Approximate number of milliseconds to wait to retrieve
+#'   updated data frame. This can be a numeric value, or a function that returns
+#'   a numeric value.
+#' @param session The user session to associate this file reader with, or NULL if
+#'   none. If non-null, the reader will automatically stop when the session ends.
+#'
+#' @importFrom shiny reactivePoll
+#' @importFrom dplyr select
+#' @importFrom dplyr filter
+#' @importFrom dplyr distinct
+#' @importFrom dplyr pull
+#'
+#' @export
+reactiveSpark <- function(x,
```

Comment:
I see now why the name might not want to have "stream" in it (b/c it's used for not just streams). Maybe `reactive_spark_df()` or `reactive_sdf()` ?